packages/cli/src/commands/identity/identifier.ts

```diff
@@ -0,0 +1,84 @@
+import { OdisUtils } from '@celo/contractkit'
+import { AuthSigner, ServiceContext } from '@celo/contractkit/lib/identity/odis/query'
+import { flags as oFlags } from '@oclif/command'
+import { cli } from 'cli-ux'
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { printValueMap } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+export default class IdentifierQuery extends BaseCommand {
+  static description =
+    'Queries ODIS for the on-chain identifier and pepper corresponding to a given phone number.'
```

Comment:
We should also add a description for the identity commands for when someone just runs `celocli`. Looks like it was missing:
https://github.com/celo-org/celo-monorepo/blob/alecps/odisCli/packages/cli/package.json#L106